OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/identity/IdentityModelStore.kt

```diff
@@ -3,7 +3,15 @@ package com.onesignal.user.internal.identity
 import com.onesignal.common.modeling.SimpleModelStore
 import com.onesignal.common.modeling.SingletonModelStore
 import com.onesignal.core.internal.preferences.IPreferencesService
+import com.onesignal.user.internal.backend.IdentityConstants
 
 open class IdentityModelStore(prefs: IPreferencesService) : SingletonModelStore<IdentityModel>(
     SimpleModelStore({ IdentityModel() }, "identity", prefs),
-)
+) {
+    fun invalidateJwt() {
+        model.setStringProperty(
```

Comment:
This will add `jwt_token` as an alias on the Identity Model and get enqueued as a set-alias operation. If you look at `IdentityModelStoreListener`, any changes are detected as aliases.